src/components/clusters/Clusters.tsx

```diff
@@ -124,7 +127,7 @@ const Clusters: React.FC = () => {
   return (
     <>
       {body}
-      {isModalOpen && <NewClusterModal closeModal={closeModal} />}
+      {isModalOpen && <NewClusterModal closeModal={closeModal} history={history} />}
```

Comment:
@rawagner, do we have this in UHC?

Reply:
commented here https://github.com/mareklibra/facet-lib/pull/27/files#r444028129